src/main.py

```diff
@@ -42,15 +43,18 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 parser = argparse.ArgumentParser()
 group = parser.add_mutually_exclusive_group()
-group.add_argument("-w", "--website", action=SingleScrapAction, help="Key of the website to scrap")
+group.add_argument("-w", "--website", default='olx', action=SingleScrapAction, help="Key of the website to scrap")
 group.add_argument("-a", "--all", action='store_true', help="Flag to scrap all websites")
+parser.add_argument("--create-database", action='store_true', help="Create database")
```

Comment:
Please add info that this mode is to create db only and it doesn't run actual scrapping

Reply:
done